### DISCUSSION TITLE
Display option "real calendar"

### DISCUSSION THREAD

**Discussion by Some1Elsewhere** [totaldebug/atomic-calendar-revive#1491] [Feature Requests]
Is there a way to display the calendar like a real one? With 2 or 4 weeks showing every day in cells but with the _actual_ events within the cell? 

I also don't mind a css option but I'm no coder, I don't know how to achieve that.

![image](https://github.com/totaldebug/atomic-calendar-revive/assets/53462922/0040f01d-eaf3-40c8-83a0-43ad4a4ff892)

**Comment by marksie1988** [comment]
This has been added to the latest beta, it will be released soon
